docker/test/integration/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:18.04
 RUN echo "deb [trusted=yes] http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" >> /etc/apt/sources.list
 
 RUN apt-get update \
-    && env DEBIAN_FRONTEND=noninteractive apt-get -y install tzdata python llvm-6.0 llvm-6.0-dev libreadline-dev libicu-dev bsdutils llvm-8 \
+    && env DEBIAN_FRONTEND=noninteractive apt-get -y install tzdata python llvm-6.0 llvm-6.0-dev libreadline-dev libicu-dev bsdutils llvm-8 curl libcurl4 libcurl4-openssl-dev \
```

Comment:
Why do we need `libcurl4 libcurl4-openssl-dev`.
The `clickhouse` binary must not have any shared library dependencies except libc.
We must not have any new build dependencies except sources in `contrib`.

Reply:
AWS SDK uses curl as http client on Linux and MacOS by default. It can be replaced with custom implementation in AWS options. I leave curl here to get early feedback about PR. What do you think about building and including curl to CH binary for now? Replacing curl with poco in AWS can take much time for investigation and implementing.

Reply:
It's in question. If we will add curl to contrib and it will pass all sanitizer tests, it should be Ok.

PS. I have heard some suspicious things about curl library (don't remember exactly, something about thread safety, issues with locking, global vars) but I have never used it, so maybe this info is just a rumor.

Reply:
Need to check for:
- the history of commits in Arcadia;
- search in internal task tracker in CONTRIB, DEVTOOLS and IGNIETFERRO queues for all the info.

Reply:
Need to check for vulnerabilities with redirects to internal hosts.

Reply:
I've checked them and found nothing suspicious.